docs/csharp/language-reference/builtin-types/struct.md

```diff
@@ -0,0 +1,56 @@
+---
+title: "Structure types - C# reference"
+ms.date: 02/20/2020
+f1_keywords: 
+  - "struct_CSharpKeyword"
+helpviewer_keywords: 
+  - "struct keyword [C#]"
+  - "structs [C#], struct keyword"
+ms.assetid: ff3dd9b7-dc93-4720-8855-ef5558f65c7c
+---
+# Structure types (C# reference)
+
+A *structure type* (or *struct type*) is a [value type](value-types.md) that encapsulates data and related functionality. As the following example shows, you use the `struct` keyword to define a structure type:
+
+[!code-csharp[struct example](~/samples/csharp/language-reference/builtin-types/StructType.cs#StructExample)]
+
+Because a structure type is a value type, a variable of a structure type contains an instance of the type. That is called value semantics. By default, variables are copied on assignment, passing an argument to a method, or returning a method result. In the case of a structure-type variable, an instance of the type is copied (for the examples, see [Value types](value-types.md)). In high-performance scenarios that involve large structure types, you can avoid value copying and pass around an alias to a value-type variable. For information about how to do that, see [Write safe and efficient C# code](../../write-safe-efficient-code.md).
+
+Typically, you define a structure type when you need to represent a small data structure that has value semantics. For example, .NET uses structure types to represent a number (both [integer](integral-numeric-types.md) and [real](floating-point-numeric-types.md)), a [Boolean value](bool.md), a [Unicode character](char.md), a [time instance](xref:System.DateTime). If you need a type with reference semantics (that is, a variable of the type contains a reference to an instance of that type, not the instance itself), define a [class](../keywords/class.md).
```

Comment:
I think we should emphasize the point in the first sentence more: Structs are when the behavior of the type is secondary to its data. (On the other hand, classes are primarily about behavior, with data that support the behavior.)